src/python/pants/bin/goal_runner.py

```diff
@@ -66,13 +66,13 @@ def _determine_v1_goals(self, options: Options) -> List[Goal]:
     v1_goals, ambiguous_goals, _ = options.goals_by_version
     return [Goal.by_name(goal) for goal in v1_goals + ambiguous_goals]
 
-  def _roots_to_targets(self, build_graph: LegacyBuildGraph, target_roots: TargetRoots):
+  def _address_specs_to_targets(self, build_graph: LegacyBuildGraph, address_specs: AddressSpecs):
```

Comment:
does this function need a return type?

Reply:
It'd be good to have one, but I have no idea what it would be. This is V1 code so I didn't spend much time trying to figure it out.